acceptance/lib/helper.rb

```diff
@@ -316,7 +316,46 @@ def hup_server(host = master, timeout = 30)
       timeout = timeout - sleeptime
     end
   end
-  
+
+  # Update a single settig in trapperkeeper config file using hocon
+  # and the Puppet DSL.
+  # Intent is that this can be used as is or as a building block in
+  # other helpers
+  #
+  # On the server specified in hocon_host, which defaults to master,
+  # update the trapperkeeper config file specified in hocon_file_path,
+  # adding/changing the setting specified in hocon_setting to the value
+  # specified in hocon_value
+  def apply_one_hocon_setting(hocon_host,
+                              hocon_file_path, 
+                              hocon_setting,
+                              hocon_value)
+    hocon_manifest =<<-EOF
```

Comment:
Your previously deployed pattern where you remove the unneeded white space in the here doc with a gsub is a good one.